src/node/server/serverPluginCss.ts

```diff
@@ -79,7 +79,6 @@ export const cssPlugin: ServerPlugin = ({ root, app, watcher, resolver }) => {
       // it cannot be handled as normal css because the js exports may change
       if (filePath.endsWith('.module.css')) {
         moduleCssUpdate(filePath)
-        return
       }
```

Comment:
This is a fix. When A improt B.B import C and C has changed. let it can bubbling with C -> B -> A.